app/Http/Controllers/CandidateController.php

```diff
@@ -132,10 +135,12 @@ public function store(Request $request, Election $election)
      * @param Candidate $candidate
      * @return Candidate
      */
-    public function show(Election $election, Candidate $candidate)
+    public function show($electionId, $candidateId)
```

Comment:
Here is another switch from type-hinting to manual lookup in the same controller.